azurerm/internal/services/azurestackhci/validate/hci_cluster.go

```diff
@@ -0,0 +1,37 @@
+package validate
```

Comment:
This also wants to be auto generated.
Once you opt in the codegen for ID validate, you will need to move the `HCIClusterName()` to a new file named *hci_cluster_name.go*.

Reply:
Could you share me an example? Thanks.

Reply:
updated